<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Boanc Ionut-Silviu</author>
 <class>settings_window</class>
 <widget class="QWidget" name="settings_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>960</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Vitesco</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CIIT</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
border-radius: 12px;
padding-left: 16px;
padding-right: 16px;
padding-top: 8 px;
padding-bottom: 8 px;
}

QRadioButton {
	font: 12pt Roboto;
	color: rgb(158, 174, 174);
    spacing: 10px;
}

QRadioButton::indicator {
    width: 16px;
    height: 16px;
	background-color: rgb(158, 174, 174);
	border-radius: 8px;
}

QRadioButton#rb_low_input::indicator:checked,
 QRadioButton#rb_low_other::indicator:checked{
   
	background-color: rgb(64, 130, 100);
}

QRadioButton#rb_normal_input::indicator:checked,
 QRadioButton#rb_normal_other::indicator:checked {
   
	background-color: rgb(249, 248, 113);
}

QRadioButton#rb_high_input::indicator:checked,
 QRadioButton#rb_high_other::indicator:checked {
   background-color: rgb(138, 29, 0);
}
</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="spacing">
    <number>8</number>
   </property>
   <item row="3" column="1">
    <layout class="QVBoxLayout" name="vl_contents">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="leftMargin">
      <number>24</number>
     </property>
     <property name="topMargin">
      <number>24</number>
     </property>
     <property name="rightMargin">
      <number>24</number>
     </property>
     <property name="bottomMargin">
      <number>48</number>
     </property>
     <item alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
      <widget class="QLabel" name="lb_overview_title">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(226, 220, 220);</string>
       </property>
       <property name="text">
        <string>Overview Page Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="fl_overview">
       <property name="horizontalSpacing">
        <number>24</number>
       </property>
       <property name="verticalSpacing">
        <number>24</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_limit_sensitivity">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(226, 220, 220);</string>
         </property>
         <property name="text">
          <string>Limits sensitivity level</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sb_limit_sensitivity">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(226, 220, 220);</string>
         </property>
         <property name="wrapping">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
         </property>
         <property name="accelerated">
          <bool>true</bool>
         </property>
         <property name="suffix">
          <string> %</string>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lb_advanced_overview_title">
       <property name="font">
        <font>
         <family>Roboto</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(226, 220, 220);</string>
       </property>
       <property name="text">
        <string>Advanced Overview Page Settings</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="fl_advanced">
       <property name="horizontalSpacing">
        <number>24</number>
       </property>
       <property name="verticalSpacing">
        <number>24</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lb_timeframe">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(226, 220, 220);</string>
         </property>
         <property name="text">
          <string>Timeframe of produced parts: </string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="sb_timeframe">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(226, 220, 220);</string>
         </property>
         <property name="buttonSymbols">
          <enum>QAbstractSpinBox::ButtonSymbols::UpDownArrows</enum>
         </property>
         <property name="suffix">
          <string> hours</string>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lb_input_sensitivity">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(226, 220, 220);</string>
         </property>
         <property name="text">
          <string>Input parts outlier detection sensitivity:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lb_other_sensitivity">
         <property name="font">
          <font>
           <family>Roboto</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(226, 220, 220);</string>
         </property>
         <property name="text">
          <string>Other parts outlier detection sensitivity:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="gb_input_sensitivity">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border-radius: 16px;	
	border-style: solid;
    border-width: 2px;
	border-color:  rgb(64, 130, 100);
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rb_low_input">
            <property name="font">
             <font>
              <family>Roboto</family>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_normal_input">
            <property name="text">
             <string>Normal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_high_input">
            <property name="text">
             <string>High</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QGroupBox" name="gb_other_sensitivity">
         <property name="styleSheet">
          <string notr="true">QGroupBox {
    border-radius: 16px;	
	border-style: solid;
    border-width: 2px;
	border-color:  rgb(64, 130, 100);
}</string>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>16</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QRadioButton" name="rb_low_other">
            <property name="text">
             <string>Low</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_normal_other">
            <property name="text">
             <string>Normal</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="rb_high_other">
            <property name="text">
             <string>High</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="3" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop">
    <widget class="QLabel" name="lb_title">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>32</pointsize>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
	color: rgb(226, 220, 220);
}</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <spacer name="vs_bottom">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="12" column="1" alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
    <widget class="QPushButton" name="pb_save">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>64</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>128</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	background-color: rgb(64, 130, 100);
	color: rgb(226, 220, 220);
    }
    QPushButton:hover {
	background-color: rgb(51, 104, 79);
    }

    QPushButton:pressed {
	
	background-color: rgb(36, 72, 55);
    }
QPushButton:disabled{
	
	background-color: rgba(64, 130, 100, 100);
    }</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" alignment="Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop">
    <widget class="QPushButton" name="pb_back">
     <property name="font">
      <font>
       <family>Roboto</family>
       <pointsize>10</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
	border-radius: 12px;
	padding-left: 8px;
	padding-right: 16px;
	padding-top: 8 px;
	padding-bottom: 8 px;
	color: rgb(226, 220, 220);
	background-color: rgb(56, 76, 83);
    }

QPushButton:hover {
	background-color: rgb(50, 68, 74);
}

QPushButton:pressed {
	
	background-color: rgb(45, 61, 66);
}</string>
     </property>
     <property name="text">
      <string>Go Back</string>
     </property>
     <property name="icon">
      <iconset resource="../icons/icons.qrc">
       <normaloff>:/Icons/arrow-left.svg</normaloff>:/Icons/arrow-left.svg</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pb_back</tabstop>
  <tabstop>pb_save</tabstop>
  <tabstop>rb_low_input</tabstop>
  <tabstop>sb_timeframe</tabstop>
  <tabstop>sb_limit_sensitivity</tabstop>
  <tabstop>rb_normal_input</tabstop>
  <tabstop>rb_high_input</tabstop>
  <tabstop>rb_low_other</tabstop>
  <tabstop>rb_normal_other</tabstop>
  <tabstop>rb_high_other</tabstop>
 </tabstops>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
 <designerdata>
  <property name="gridDeltaX">
   <number>8</number>
  </property>
  <property name="gridDeltaY">
   <number>8</number>
  </property>
  <property name="gridSnapX">
   <bool>true</bool>
  </property>
  <property name="gridSnapY">
   <bool>true</bool>
  </property>
  <property name="gridVisible">
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
